roles/internal/webserver-app/tasks/solr.yml

```diff
@@ -3,6 +3,11 @@
 
 # Only tasks related to configuring search_api_solr are put here
 # Solr server configuration should go into the solr playbook: https://github.com/Islandora-Devops/claw-playbook/blob/dev/solr.yml
+- name: Add a patch for symfony-event-dispatcher per the INSTALL.md docs on search_api_solr:3.8
+  composer:
+    command: require
+    arguments: symfony/event-dispatcher:'4.3.4 as 3.4.99' drupal/search_api_solr
+    working_dir: "/var/www/html/drupal"
```

Comment:
Can be `{{ drupal_composer_install_dir }}`
https://github.com/Islandora-Devops/islandora-playbook/blob/dev/inventory/vagrant/group_vars/webserver/drupal.yml#L4

Reply:
Thanks, I'll fix that

Reply:
fixed with 
036287f